packages/remediations/src/steps/selectPlaybook.js

```diff
@@ -27,6 +27,10 @@ const SelectPlaybook = (props) => {
     const { input } = useFieldApi(props);
     const formOptions = useFormApi();
     const values = formOptions.getState().values;
+    const allSystems = [ ...new Set(issues.reduce((acc, curr) => [
```

Comment:
You have multiple `[... new Set(...)]` calls. I think we can create a helper function called something like `dedupeArray`.